Tests/Unit/Http/RuleMatcherTest.php

```diff
@@ -45,4 +45,16 @@ public function testMatchResponse()
         $this->assertFalse($ruleMatcher->matches(new Request(), new Response('', 100)));
         $this->assertTrue($ruleMatcher->matches(new Request(), new Response('', 500)));
     }
+
+    public function testMatchResponseRequest()
+    {
+        $ruleMatcher = new RuleMatcher(new RequestMatcher(), array('match_response' => 'response.getStatusCode() == 201 & request.getMethod() == \'PUT\''));
```

Comment:
you're absolutely right; duplicating my match rule, restricting it to a ``PUT`` method and a ``201`` response code does exactly that… 

thanks for clarifying, must have been asleep yesterday :sleeping: